waku/v2/peermanager/peer_manager.go

```diff
@@ -73,40 +94,112 @@ func (pm *PeerManager) connectivityLoop(ctx context.Context) {
 		case <-ctx.Done():
 			return
 		case <-t.C:
-			pm.pruneInRelayConns()
+			pm.connectToRelayPeers()
 		}
 	}
 }
 
-func (pm *PeerManager) pruneInRelayConns() {
+// GroupPeersByDirection returns all the connected peers in peer store grouped by Inbound or outBound direction
+func (pm *PeerManager) GroupPeersByDirection() (inPeers peer.IDSlice, outPeers peer.IDSlice, err error) {
+
+	for _, p := range pm.host.Network().Peers() {
+		direction, err := pm.host.Peerstore().(wps.WakuPeerstore).Direction(p)
+		if err == nil {
+			if direction == network.DirInbound {
+				inPeers = append(inPeers, p)
+			} else if direction == network.DirOutbound {
+				outPeers = append(outPeers, p)
+			}
+		}
+	}
+	return inPeers, outPeers, nil
+}
 
-	var inRelayPeers peer.IDSlice
+func (pm *PeerManager) getRelayPeers() (inRelayPeers peer.IDSlice, outRelayPeers peer.IDSlice) {
 	//Group peers by their connected direction inbound or outbound.
-	inPeers, outPeers, err := pm.host.Peerstore().(wps.WakuPeerstore).GroupPeersByDirection()
+	inPeers, outPeers, err := pm.GroupPeersByDirection()
 	if err != nil {
 		return
 	}
-	pm.logger.Info("Number of peers connected", zap.Int("inPeers", inPeers.Len()), zap.Int("outPeers", outPeers.Len()))
+	pm.logger.Info("Number of peers connected", zap.Int("inPeers", inPeers.Len()),
```

Comment:
Should this be  `Info` or `Debug` level?

Reply:
Wanted to keep it info for now, same has been done in nwaku.